koku/api/report/gcp/provider_map.py

```diff
@@ -61,8 +66,9 @@ def __init__(self, provider, report_type):
                         {"field": "project_name", "operation": "icontains", "composition_key": "project_filter"},
                         {"field": "project_id", "operation": "icontains", "composition_key": "project_filter"},
                     ],
+                    "instance_type": {"field": "instance_type", "operation": "icontains"},
```

Comment:
Other source tpyes don't have a instance_type filter, so I think maybe we don't need this?